src/codegen_call_thread.cpp

```
@@ -119,224 +134,227 @@ llvm::Value *CodeGen::emitThreadSpawn(const CallExpr &e) {
+            envFields.push_back(cg.i8Ty_);
         else
             for (const auto &[_, alloca] : captures)
                 envFields.push_back(alloca->getAllocatedType());
-        llvm::StructType *envTy = llvm::StructType::get(*ctx_, envFields);
+        llvm::StructType *envTy = llvm::StructType::get(*cg.ctx_, envFields);
 
-        llvm::FunctionCallee mallocFn = getStdlibMalloc();
-        const llvm::DataLayout &dl = mod_->getDataLayout();
+        llvm::FunctionCallee mallocFn = cg.getStdlibMalloc();
+        const llvm::DataLayout &dl = cg.mod_->getDataLayout();
         uint64_t envSize = std::max<uint64_t>(1, dl.getTypeAllocSize(envTy));
```

Clang (14.0.6)

[warning] variable 'envSize' is not initialized

(cppcoreguidelines-init-variables)